test/README.md

```diff
@@ -0,0 +1,158 @@
+Running DMD's test suite
+-------------------------
+
+This is the documentation and guide for DMD's test suite.
+Maybe you are looking for the [contributing guide](../CONTRIBUTING.md) too?
+
+Quick guide
+-----------
+
+### Run all tests
+
+```sh
+make -j10
+```
+
+Note: the exact parallelism parameter `-j` depends on your system,
+but if you have enough memory, a high `-j` allows a significant speed-up.
```

Comment:
FWIW on my machine i7 4th Gen with 16GB RAM I usually get the best results with `-j20`.
Running all `fail_compilation` tests takes merely 7s (with `-j10` it's 10s)